docs/getting-setup.md

```diff
@@ -98,38 +98,6 @@ Navigate to `about:config` and accept any warning message. Then search for the f
 
 Close firefox and re-open it with the `firefox-bin` command.
 
-#### Firefox GCLI
```

Comment:
This is from your previous change.  You'll want to learn to use feature branches moving forward.

Reply:
I just read the documentation just after submitting the last PR. I am trying to revert the last commit and change it to feature branch! Will that be alright? I am figuring out how to do the same. Can I get some time so that I can fix it?